obal/data/roles/git_branch_and_commit/tasks/main.yml

```diff
@@ -6,21 +6,6 @@
   changed_when: false
   register: updated_version
 
-- name: 'Get current git branch'
-  command: "git rev-parse --abbrev-ref HEAD"
-  args:
-    chdir: "{{ inventory_dir }}"
-  changed_when: false
-  register: git_branch
-  run_once: true
-
-- name: 'Create git branch'
-  command: "git checkout -b {{ git_branch.stdout }}-update-{{ inventory_hostname }}-{{ updated_version.stdout }}"
-  args:
-    chdir: "{{ inventory_dir }}"
-  run_once: true
-  changed_when: true
```

Comment:
What do you think about moving this idea to the playbook (https://github.com/theforeman/obal/blob/master/obal/data/playbooks/update/update.yaml) and only performing it once? For single packages, I find it very useful to:

```
git checkout rpm/develop
git reset origin/rpm/develop
obal update <package> --commit
git push ehelms <current branch>
```